linera-rpc/proto/rpc.proto

```diff
@@ -141,17 +141,22 @@ message BlockProposal {
 
 // A certified statement from the committee, without the value.
 message LiteCertificate {
+  // The ID of the chain the value belongs to.
+  ChainId chain_id = 10;
+
   // The certified value's hash
   bytes hash = 1;
 
-  // The ID of the chain the value belongs to.
-  ChainId chain_id = 2;
-
   // Signatures on the value
-  bytes signatures = 3;
+  bytes signatures = 2;
+
+  // Wait until all outgoing cross-chain messages from this certificate have
+  // been received by the target chains.
+  bool wait_for_outgoing_messages = 3;
 }
 
-// A certified statement from the committee.
+// A certified statement from the committee, together with other certificates
+// required for execution.
 message Certificate {
   // The ID of the chain (used for routing).
   ChainId chain_id = 10;
```

Comment:
Here and in other places, can you normalize the grpc field numbers to start from `1`? (It's on me: I forgot to finalize local changes.)